migrations/2024-08-20-112035_add-profile-id-to-txn-tables/up.sql

```diff
@@ -0,0 +1,19 @@
+-- Your SQL goes here
+ALTER TABLE payment_attempt
+ADD COLUMN IF NOT EXISTS profile_id VARCHAR(64) NOT NULL DEFAULT 'default_profile';
```

Comment:
This will lead to the table having invalid key reference. Is that fine? Can you also include the data backfill queries in the upcoming PRs?

Reply:
this isn't a FK reference,
I'll add the backfill queries in the up.sql as well